locales-lt-LT.xml

```diff
@@ -169,7 +168,7 @@
     <term name="issue" form="short">nr.</term>
     <term name="line" form="short">l.</term>
     <term name="note" form="short">n.</term>
```

Comment:
pstb.? I actually have no idea what the proper abbreviation should be, but this doesn't match the long form.